### DISCUSSION TITLE
[OTHER]: Small question (server side / client side)

### DISCUSSION THREAD

**Discussion by chmonyasik** [AnOpenSauceDev/FastRandom#48] [Q&A] [answered]
### Does an issue already cover this?

- [X] I have searched for any existing issues, and didn't find any.

### Issue description.

small question, does it make sense to use it if it is on the client but not on the server side 🙃

**Comment by Steveplays28** [comment, marked as answer]
Hi, Faster Random can be used on both the client and the server, but you're not required to have it on both sides at the same time.
Hope that answers your question!

**Reply by chmonyasik** [reply]
Thank you!
